test/Pomelo.EntityFrameworkCore.MySql.Functional/project.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+  "dependencies": {
+    "dotnet-test-mstest": "1.1.1-preview",
+    "MSTest.TestFramework": "1.0.1-preview",
```

Comment:
I will try it!  Shouldn't be an issue if XUnit still works with Dependency Injection